packages/react/src/Button/Button.client.tsx

```diff
@@ -0,0 +1,52 @@
+import "@axa-fr/design-system-css/dist/Button/Button.client.scss";
+import { cx } from "classix";
+
+import { ComponentPropsWithoutRef, PropsWithChildren } from "react";
+
+export enum Variants {
+  primary = "primary",
+  business = "primary-business",
+  secondary = "secondary",
+  secondaryError = "secondary-error",
+  tertiary = "tertiary",
+  ghost = "ghost",
+}
+
+type ButtonProps = {
+  variant?: Variants;
+  iconLeft?: string;
+  iconRight?: string;
+} & ComponentPropsWithoutRef<"button">;
+
+export const ButtonClient = ({
+  children,
+  className,
+  ...args
+}: PropsWithChildren<ButtonProps>) => {
+  const disabled = args.disabled ? `af-btn-client--disabled` : "";
+  const variant = args.variant
+    ? `af-btn-client--${args.variant.toString()}`
+    : "";
+  const iconLeft = args.iconLeft ? `af-btn-client--hasiconLeft` : undefined;
+  const iconRight = args.iconRight ? `af-btn-client--hasiconRight` : undefined;
+  return (
+    <button
+      className={cx(
+        "af-btn-client",
+        variant,
+        disabled,
+        iconLeft,
+        iconRight,
+        className,
+      )}
+      type="button"
+      {...args}
+    >
+      {iconLeft && <span className={`glyphicon glyphicon-${args.iconLeft}`} />}
+      {children}
+      {iconRight && (
+        <span className={`glyphicon glyphicon-${args.iconRight}`} />
+      )}
+    </button>
+  );
+};
```

Comment:
suite à mon commentaire sur la props en React Node je ferais comme ceci
```suggestion
type ButtonProps = {
  variant?: Variants;
  iconLeft?: ReactNode;
  iconRight?: ReactNode;
} & ComponentPropsWithoutRef<"button">;

export const ButtonClient = ({
  children,
  className,
  disabled,
  variant,
  iconLeft,
  iconRight,
  ...props
}: PropsWithChildren<ButtonProps>) => {
  return (
    <button
      className={cx(
        "af-btn-client",
        variant && `af-btn-client--${args.variant.toString()}`,
        disabled && 'af-btn-client--disabledé,
        iconLeft && 'af-btn-client--hasiconLeft',
        iconRigh && 'af-btn-client--hasiconRight't,
        className,
      )}
      type="button"
      disabled={disabled}
      {...props}
    >
      {iconLeft}
      {children}
      {iconRight}
    </button>
  );
};
```